wagtail/images/static_src/wagtailimages/js/focal-point-chooser.js

```diff
@@ -30,6 +30,20 @@ function setupJcrop(image, original, focalPointOriginal, fields) {
       // Set alt="" on the image so its src is not read out loud to screen reader users.
       var $holderImage = $('img', jcropapi.ui.holder);
       $holderImage.attr('alt', '');
+
+      // Create a label tag for the input for accessibility.
+      const label = document.createElement('label');
+
+      label.setAttribute('for', 'jcrop');
+      label.classList.add('className', 'visuallyhidden');
+      label.innerHTML = 'Cropping Tool';
+
+      var holder = document.getElementsByClassName('jcrop-holder');
+      holder[0].appendChild(label);
+
+      // Set the name attribut on the input corresponding to the label.
+      var $holderInput = $('input', jcropapi.ui.holder);
```

Comment:
as above, the name will not be suitable.
Also - adjusting the name may cause some odd issues for form submission.